ArduSub/sensors.cpp

```diff
@@ -4,6 +4,11 @@
 void Sub::read_barometer()
 {
     barometer.update();
+    // If we are reading a positive altitude, the sensor needs calibration
+    // Even a few meters above the water we should have no significant depth reading
+    if(barometer.get_altitude() > 0) {
```

Comment:
Let's add a check that we are not armed so nothing funny happens in action.

Reply:
Done

Reply:
I had missed a `!`, updated.